src/App.js

```diff
@@ -1,12 +1,33 @@
+import { BrowserRouter, Route, Routes } from "react-router-dom";
 import "./App.css";
-import Main from "./Main";
-import Nav from "./Nav";
+import HomePage from "./components/HomePage";
+import Nav from "./components/Nav";
+import BookPage from "./components/BookPage";
+import ClaimedBooks from "./components/ClaimedBooks";
+import { useEffect, useState } from "react";
 
 function App() {
+  const [books, setBooks] = useState([])
+  useEffect(() => {
```

Comment:
Move this into the homepage since it doesn't need to be shared with any other pages